### DISCUSSION TITLE
Payment vs TransactionItem model

### DISCUSSION THREAD

**Discussion by PramodMandati** [saleor/saleor#11802] [General] [answered]
In most of the cases i saw the Payment models is being used (Example: in the method can_refund in order model). But when i do the payment from the stripe only TransactionItem and TransactionEvent row's are being created in the database. Can someone explain the difference between these two models?

**Comment by timuric** [comment, marked as answer]
@PramodMandati transaction API allows creating multiple payments on individual order, for example you could capture credit card and cash on delivery, and have separate refunds for each transaction. Basically, transactions API is the new flexible way of working with payments in Saleor. We are working on the new API that would be replacing existing `checkoutComplete` mutation with an alternative that would use transactions instead. Eventually, Payment models would be deprecated.

Currently, transactions are used by payment apps, and we recommend using apps instead of payment plugins
https://docs.saleor.io/docs/3.x/developer/checkout#finalizing-checkout-with-saleor-app

**Reply by PramodMandati** [reply]
@timuric If Payment model is going to deprecated then the manually marking a payment as paid is not required? Because in the mutation order_mark_as_paid a Payment model row is being created.
Can we send exact authorised amount in the create_transaction mutataion in the webhook api stripe.ts? so don't need to call the order_mark_as_paid.

**Reply by timuric** [reply]
@PramodMandati yes you can pass authorized amount https://docs.saleor.io/docs/3.x/developer/checkout#creating-transaction
